### PR TITLE
show asset address & logo in dropdown

### PR DESCRIPTION
**Pull request by 0xvgd**
image

https://github.com/polymorpher/otc-country/issues/4
https://github.com/polymorpher/otc-country/issues/5
https://github.com/polymorpher/otc-country/issues/6
https://github.com/polymorpher/otc-country/issues/7

- add validation to check if source and dest asset are the same
- fix bug that shows domain owner is required
- improve switch network UI so user can't continue

image

- remove spinner as already showing in domain input

### CODE REVIEW

**Comment by polymorpher** [comment]
Depegged 1ETH logo can be same as 1ETH.

Let's add a grayscale filter to all depegged assets

WONE / Harmony logo svg

harmony

File: app/src/components/ChainDetector.tsx [hunk 1]
```diff
@@ -10,21 +10,24 @@ const ChainDetector = (): React.JSX.Element | null => {
...
-      <Spacer />
-      <Button colorScheme="yellow" onClick={() => switchNetwork?.()} isLoading={isLoading} loadingText="Switch">
-        Switch
-      </Button>
-    </Alert>
+    <Drawer placement="bottom" isOpen={isConnected && currentChain?.id !== chain.id} onClose={() => true}>
+      <DrawerOverlay />
+      <DrawerContent>
+        <DrawerHeader borderBottomWidth='1px'>
+          Your metamask is on a wrong chain
```
Comment (polymorpher) [inline app/src/components/ChainDetector.tsx]: Wallet connected to unsupported network

**Comment by polymorpher** [comment]
depegged assets should not be destination asset. See lists: https://github.com/polymorpher/otc-country/issues/4

**Comment by polymorpher** [comment]
Looks good overall.

- [x] if domain owner equals connected wallet address, show it as `...<address>... (you)` 
- [x] more vertical spacing between each input group
- [x] fixed and estimated values should all always show 2 decimal points
- [x] allow any number in deposit amount, but show error message (red alert box below it) if user's wallet does not have sufficient funds
- [x] show estimated exchange rate (source estimated value / destination estimated value) 
- [x] less opacity for unselected commission rate button/input box
- [x] show % symbol in the custom commission rate input box
- [x] remove all marks in the slider, only keep two ends
- [x] make sure the slider floating label is wide enough - show text in one line
- [x] bigger "Create" button
- [x] info alert below: "Tips: You can retract the offer at any time after it is created. Other people can join your offer by depositing asset there. The domain owner earns commission if the offer is accepted by someone (who has to deposit <target asset name>)"

image

**Comment by jwst9** [comment]
image

**Comment by polymorpher** [comment]
Looks good. Let's show user balance of the select at the right (BALANCE: ...) in the next PR

**Review by polymorpher — APPROVED** [review]

**Pull request merged**